docs/user/dql/functions.rst

```diff
@@ -1448,6 +1458,55 @@ Example::
     | 4                               |
     +---------------------------------+
 
+    os> SELECT DAYOFWEEK(DATETIME('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +----------------------------------------------+
+    | DAYOFWEEK(DATETIME('2020-08-26 00:00:00'))   |
+    |----------------------------------------------|
+    | 4                                            |
+    +----------------------------------------------+
+
+    os> SELECT DAYOFWEEK(TIMESTAMP('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +-----------------------------------------------+
+    | DAYOFWEEK(TIMESTAMP('2020-08-26 00:00:00'))   |
+    |-----------------------------------------------|
+    | 4                                             |
+    +-----------------------------------------------+
+
+    os> SELECT DAY_OF_WEEK('2020-08-26')
+    fetched rows / total rows = 1/1
+    +-----------------------------+
+    | DAY_OF_WEEK('2020-08-26')   |
+    |-----------------------------|
+    | 4                           |
+    +-----------------------------+
+
+    os> SELECT DAY_OF_WEEK(DATE('2020-08-26'))
+    fetched rows / total rows = 1/1
+    +-----------------------------------+
+    | DAY_OF_WEEK(DATE('2020-08-26'))   |
+    |-----------------------------------|
+    | 4                                 |
+    +-----------------------------------+
+
+    os> SELECT DAY_OF_WEEK(DATETIME('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +------------------------------------------------+
+    | DAY_OF_WEEK(DATETIME('2020-08-26 00:00:00'))   |
+    |------------------------------------------------|
+    | 4                                              |
+    +------------------------------------------------+
+
+    os> SELECT DAY_OF_WEEK(TIMESTAMP('2020-08-26 00:00:00'))
```

Comment:
consider having both `dayofweek` and `day_of_week` on the same line to make comparisons easier to show equivalence. 

eg. `SELECT DAYOFWEEK(DATETIME('2020-08-26 00:00:00')), DAY_OF_WEEK(DATETIME('2020-08-26 00:00:00'));`

Reply:
also consider removing a lot of these from the documentation.  It's a little cumbersome and the variants are already tested in `DateTimeFunctionIT`. I'd just include one example for DAYOFWEEK and one example of DAY_OF_WEEK or combine them like Mitchell suggested.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/190/commits/e571bf04368910cfc8be87e69e2cf901bad7a46e